index.bs

```diff
@@ -109,7 +116,14 @@ Long Task timing involves the following new interfaces
 
 {{PerformanceLongTaskTiming}} extends the following attributes of {{PerformanceEntry}} interface:
 
-* The {{PerformanceEntry/name}} attribute must return {{DOMString}} for minimal frame attribution. Possible values of name are:
+* The {{PerformanceEntry/name}} attribute must return {{DOMString}} that specifies the type of long task being reported. For long tasks obtained from event loop <a>tasks</a>, it also provides minimal frame attribution.
```

Comment:
Link to "long task" here once using `[=long task=]` or `<a>long task</a>`. (No need to do it for every occurrence, but this is the first mention in this section.)

Reply:
Done